src/NUnit.Xml.TestLogger/NUnit.Xml.TestLogger.csproj

```diff
@@ -17,4 +17,11 @@
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="15.5.0" />
     <PackageReference Include="System.ValueTuple" Version="4.3.0" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Stylecop.Analyzers" Version="1.1.118">
```

Comment:
Stylecop should be updated in scripts/dependencies.props

Reply:
Ah, got it. I was surprised to see stylecop was added to these csproj files but wasn't sure where else it should have happened.